osu.Game/Overlays/Comments/VotePill.cs

```diff
@@ -62,21 +67,31 @@ private void load(OsuColour colours)
             AccentColour = borderContainer.BorderColour = sideNumber.Colour = colours.GreenLight;
             hoverLayer.Colour = Color4.Black.Opacity(0.5f);
 
-            if (api.IsLoggedIn && api.LocalUser.Value.Id != comment.UserId)
+            var ownComment = api.LocalUser.Value.Id == comment.UserId;
+
+            if (!ownComment)
                 Action = onAction;
```

Comment:
For style points, this could ideally use `Enabled`. Should save you having to expose the background box, actually, and having to do this disable logic manually.

Reply:
Updated, should be better